src/test/java/teammates/test/pageobjects/AdminEmailPage.java

```diff
@@ -20,6 +23,18 @@ public void inputRecipient(String recipient) {
         recipientBox.sendKeys(recipient);
     }
 
+    public void inputGroupRecipient(String fileName) {
+        WebElement groupReceiverListUploadBox = this.getGroupReceiverListUploadBox();
+        JavascriptExecutor js = (JavascriptExecutor) browser.driver;
+        js.executeScript("arguments[0].style.display = 'inline'", groupReceiverListUploadBox);
+        File file = new File("src/test/resources/data/" + fileName);
+        WebElement uploadGroupListFile = this.uploadGroupListFile();
+        uploadGroupListFile.sendKeys(file.getAbsolutePath());
+        js.executeScript("arguments[0].style.display = 'none'", groupReceiverListUploadBox);
```

Comment:
it is because initially, the uploadBox is hidden, so we first set it visible, upload the required field and again set it invisible.